admin/src/react-components/accounts.js

```diff
@@ -40,29 +40,143 @@ const AccountFilter = props => (
 
 export const AccountList = withStyles(styles)(
   class AccountList extends Component {
-    state = {};
+    state = {
+      emailSearch: "",
+      searching: false,
+      searchStatus: null,
+      emailCreate: "",
+      identityCreate: "",
+      creating: false,
+      createStatus: null,
+      createResults: ""
+    };
     async onAccountSearch(e) {
       e.preventDefault();
-      this.setState({ searching: true, accountSearchStatus: null });
+      this.setState({ searching: true, searchStatus: null });
       const result = await fetch("/api/v1/accounts/search", {
         method: "post",
         headers: {
           "content-type": "application/json",
           authorization: `bearer ${window.APP.store.state.credentials.token}`
         },
-        body: JSON.stringify({ email: this.state.email || "" })
+        body: JSON.stringify({ email: this.state.emailSearch || "" })
       }).then(r => r.json());
       if (result && result.data) {
         window.location = `#/accounts/${result.data[0].id}`;
       } else {
-        this.setState({ searching: false, accountSearchStatus: "Account not found" });
+        this.setState({ searching: false, searchStatus: "Account not found" });
       }
+      // Quickfix snackbar component does not always close
+      // Setting snackbar message to empty string closes
+      setTimeout(() => this.setState({ searchStatus: "" }), 5000);
```

Comment:
It's possible this gets called after the component has unmounted. Maybe clear the timeout in `componentWillUnmount`?